docs/content/sips/019-governance.md

```diff
@@ -0,0 +1,21 @@
+title = "SIP 019 - Spin Governance"
+template = "main"
+date = "2024-06-18T23:00:00Z"
+
+---
+
+Summary: Adding governance to the Spin project
+
+Owner: michelle@fermyon.com
+
+Created: June 18, 2024
+
+## Background
+
+Spin is a project now used by many organizations and individuals across the software ecosystem. In order for the Spin community to continue to rely on, build on, and integrate with Spin, there should be a public collective understanding of how the Spin project is governed and how to iterate on governance.
+
+## Proposal
+
+This SIP is a proposal to add a GOVERNANCE.md file to the root of the Spin repository. The governance structure described in the corresponding GOVERNANCE.md file aims to describe transparently a process close to how we've been doing things so far in the project and how to make changes. It formally opens up the project for maintainers from multiple organizations, defines maintainership roles, describes decision making processes and more.
```

Comment:
We should try to remember to link to the current version of `GOVERNANCE.md` so that it's easy in the future to see the exact version of the governance.md that was agreed to by this SIP (without having to crawl GitHub). Of course, we can't link to the exact version until we're sure it's final.